### PR TITLE
Bump jruby 9k to rc1

### PR DESCRIPTION
**Pull request by cpuguy83**
Signed-off-by: Brian Goff <cpuguy83@gmail.com>

### CODE REVIEW

File: library/jruby [hunk 1]
```diff
@@ -13,15 +13,15 @@ latest: git://github.com/cpuguy83/docker-jruby@882f6121c19e90e158bc86645335f8762
...
 1.7.20-onbuild: git://github.com/cpuguy83/docker-jruby@882f6121c19e90e158bc86645335f8762d26ff19 1.7/onbuild
 
-9.0.0.0: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
-9.0.0.0-jre: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
-9.0.0.0.pre2: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
-9.0.0.0.pre2-jre: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
+9.0.0.0: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
+9.0.0.0-jre: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
+9.0.0.0.rc1: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
+9.0.0.0.rc1-jre: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
```
Comment (tianon) [inline library/jruby]: Not sure where this commit ID comes from, but I think you meant d7f7b44668aeb8f192d210302aff23bfde1ab7fa (https://github.com/cpuguy83/docker-jruby/commit/d7f7b44668aeb8f192d210302aff23bfde1ab7fa).

Comment (cpuguy83) [inline library/jruby]: Forgot to push, it should be there now :)

**Comment by tianon** [comment]
https://github.com/cpuguy83/docker-jruby/compare/882f6121c19e90e158bc86645335f8762d26ff19...14113750316f06b2785a41b5bc3dd6b23510af6f

Build test of #802; 0da315005bc54c33e1800017a15b9b36cd4ca8e9 (`jruby`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/0da315005bc54c33e1800017a15b9b36cd4ca8e9/library/jruby"
$ bashbrew build "$url"
Fetching jruby (git://github.com/cpuguy83/docker-jruby) ...
Processing jruby:1.7 ...
Processing jruby:1.7.20 ...
Processing jruby:latest ...
Processing jruby:1.7-jre ...
Processing jruby:1.7.20-jre ...
Processing jruby:1.7-jdk ...
Processing jruby:1.7.20-jdk ...
Processing jruby:1.7-onbuild ...
Processing jruby:1.7.20-onbuild ...
Processing jruby:9.0.0.0 ...
Processing jruby:9.0.0.0-jre ...
Processing jruby:9.0.0.0.rc1 ...
Processing jruby:9.0.0.0.rc1-jre ...
Processing jruby:9.0.0.0-jdk ...
Processing jruby:9.0.0.0.rc1-jdk ...
Processing jruby:9.0.0.0-onbuild ...
Processing jruby:9.0.0.0.rc1-onbuild ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing jruby:1.7
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-root-password' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...failed
testing jruby:1.7-jdk
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-root-password' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...failed
testing jruby:1.7-onbuild
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-root-password' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...failed
testing jruby:9.0.0.0
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-root-password' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...failed
testing jruby:9.0.0.0-jdk
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-root-password' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...failed
testing jruby:9.0.0.0-onbuild
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-root-password' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...failed
```

What's making `ruby-bundler` fail this time, I wonder? :disappointed:

**Comment by tianon** [comment]
Here's why it's failing this time:
```diff
diff --git a/test/tests/ruby-bundler/Gemfile.lock b/test/tests/ruby-bundler/Gemfile.lock
index 2b6d524..4be58e0 100644
--- a/test/tests/ruby-bundler/Gemfile.lock
+++ b/test/tests/ruby-bundler/Gemfile.lock
@@ -28,3 +28,6 @@ DEPENDENCIES
   nokogiri
   rack (~> 1.1)
   rspec
+
+BUNDLED WITH
+   1.10.3
```

What the heck, Ruby?

**Comment by tianon** [comment]
LGTM anyhow -- let's figure out the test issue separately

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**